### ISSUE TITLE
Download URL text is wrong when version is first added.

### ISSUE THREAD

**Issue by adamk33n3r** [TechnicPack/TechnicSolder#563]
When I first add a version of a mod the Download URL text seems to be corrupt. The url of the link itself is correct because if I click on it it correctly downloads the mod. It's just the text as you can see in the image below.
![image](https://cloud.githubusercontent.com/assets/1350444/13562962/a2616d76-e40b-11e5-8853-55d308ae2dc6.png)
If I refresh the issue goes away, though.
![image](https://cloud.githubusercontent.com/assets/1350444/13563109/c65ec1a0-e40c-11e5-862a-89f2e492aa71.png)

**Comment by spannerman79** [comment]
Strange - I just installed solder fresh on a new VPS and I don't have that issue.

I am using MySQL, PHP 5.6 and Apache though

**Comment by adamk33n3r** [comment]
Hm. I'm using sqlite, php7, and nginx if that helps.

**Comment by spannerman79** [comment]
It *could* be a combination of PHP7 & ngix - *maybe*. I did notice with some of the commits in dev branch there was some work on PHP7 being done - not sure if its ready for PHP7 or not.  Someone else will have to answer that.

**Comment by spannerman79** [comment]
Hang on, your right @adamk33n3r.  It *just* happened to me as well.

So far it happens on
* Apache or nginx
* SQLite or MySQL
* PHP7 or 5.6

For the record my VPS that I just installed to is CentOS 7.

My apologies for jumping the gun on you @adamk33n3r.

**Comment by adamk33n3r** [comment]
@spannerman79 no worries :)

Mines installed on an arch box for the record.

**Comment by tankerkiller125** [comment]
I also have this issue.
* Ubuntu 14.04 LTS Server
* MySQL (Percona Cluster to be exact)
* PHP 7.0
* Nginx (Latest)

I also just tested it on an AWS VPS and saw the issue there as well.

**Comment by aschmois** [comment]
There's an extra { in code, it's a very simple fix.

https://github.com/TechnicPack/TechnicSolder/blob/master/app/views/mod/view.blade.php#L149

Change the line from:
`$("#add-row").after('<tr><td></td><td>' + data.version + '</td><td>' + data.md5 + '</td><td><a href="' + mirror_url + 'mods/{{ $mod->name }}/{{ $mod->name }}-' + data.version + '.zip" target="_blank">{mods/{{ $mod->name }}/{{ $mod->name }}-' + data.version + '.zip</a></td><td>' + data.filesize + '</td><td></td></tr>');`

to:
`$("#add-row").after('<tr><td></td><td>' + data.version + '</td><td>' + data.md5 + '</td><td><a href="' + mirror_url + 'mods/{{ $mod->name }}/{{ $mod->name }}-' + data.version + '.zip" target="_blank">' + mirror_url + 'mods/{{ $mod->name }}/{{ $mod->name }}-' + data.version + '.zip</a></td><td>' + data.filesize + '</td><td></td></tr>');`

And the issue fixes itself.

**Comment by GenPage** [comment]
Will close issue once dev is merged into master.

**Comment by Pyker** [comment]
The frontend was completely rewritten in v1.0.0 with Tailwind CSS and Alpine.js. This UI bug is no longer reproducible. Closing as fixed. If you still see this issue on v1.0.0, please reopen.
